core/includes/database/mysql/database.inc

```diff
@@ -110,6 +110,14 @@ class DatabaseConnection_mysql extends DatabaseConnection {
     }
   }
 
+  public function query($query, array $args = array(), $options = array()) {
+    $query = preg_replace('/{([^}]+)}/', '`\1`', $query);
+    // This to make Drush work
```

Comment:
Proper English + period at the end:
```suggestion
    // This gets Drush working with MySQL 8 as well.
```

We might wanna add a comment at the start of this entire block of code, to mention that it is for supporting MySQL 8.